install_config/master_node_configuration.adoc

```diff
@@ -1762,38 +1762,58 @@ $ openshift start node --config=/openshift.local.config/node-<node_hostname>/nod
 |===
 
 You can control which INFO messages are logged by setting the loglevel option in the in xref:../admin_guide/manage_nodes.adoc#modifying-nodes[node configuration files], 
-the *_/etc/origin/master/master-config.yaml_* file. Configuring the logs to collect all messages can lead to large logs that are difficult to interpret and can take up excessive space. Collecting all messages should only be used in debug situations.
+the *_/etc/sysconfig/atomic-openshift-node_* file. Configuring the logs to collect all messages can lead to large logs that are difficult to interpret and can take up excessive space. Collecting all messages should only be used in debug situations.
 
 [NOTE]
 ====
 Messages with FATAL, ERROR, WARNING and some INFO severities appear in the logs regardless of the log configuration.
 ====
 
-You can view logs for the master or the node system using the following command:
+*View node logs*
+
+To view logs for the node system, use the following command:
 
 ----
 # journalctl -r -u <journal_name>
 ----
 
+*View master logs*
+
+To view logs for the master, use the following command:
+
+----
+# /usr/local/bin/master-logs <component> <container name> <1>
+----
+
 Use the `-r` option to show the newest entries first.
 
 For example:
 
 ----
-# master-logs controllers controllers
-# master-logs api api
+# /usr/local/bin/master-logs controllers controllers
+# /usr/local/bin/master-logs api api
 # journalctl -r -u atomic-openshift-node.service
 ----
 
+*Redirect master log to a file*
+
+To redirect the output of master log into a file, use the following command:
+
+----
+master-logs api api 2> file
+----
+
+*Change the level of logging*
+
 To change the logging level:
 
-. Edit the *_/etc/sysconfig/atomic-openshift-master_* file for the master or *_/etc/sysconfig/atomic-openshift-node_* file for the nodes.
+. Edit the *_/etc/origin/master/master.env_* file for the master or *_/etc/sysconfig/atomic-openshift-node_* file for the nodes.
 . Enter a value from the *Log Level Options* table above in the `OPTIONS=--loglevel=` field.
 +
 For example:
 +
 ----
-OPTIONS=--loglevel=4
+DEBUG_LOGLEVEL=--loglevel=4
```

Comment:
Should be `DEBUG_LOGLEVEL=4`, no --loglevel


> above in the `OPTIONS=--loglevel=` field

Need change this sentence too